.gitignore

```diff
@@ -11,6 +11,7 @@ default/
 *.user*
 *.creator.*
 *.autosave
+build/ 
```

Comment:
Is it okay to add this? When I run Qt Creator locally it seems to create this directory

Reply:
I have my Qt Creator set up with a different path, but since I guess this is the default, it's alright to add this.